cli/internal/providers/datacatalog/importremote/provider/trackingplan.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 const (
-	TrackingPlanScope         = "trackingplan"
-	TrackingPlansRelativePath = "trackingplans"
+	trackingPlanFileNameScope = "file-name-trackingplan"
+	trackingPlansRelativePath = "trackingplans"
```

Comment:
Making the non-exported fields as smallcased and also updating the fieldname to be named properly to trackingPlanFileNameScope as this value is used when generating different file names for the trackingplan during the import